Packs/MapPattern/ReleaseNotes/1_0_6.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+- **MapPattern**
+- Fixed an issue where it returns successfully when a syntax error occurred.
```

Comment:
```suggestion
- Fixed an issue where the script returned a success message when a syntax error occurred.
```

Reply:
@YuvHayun OK, I've fixed it.

Reply:
My bad, I was trying to give a different fix suggestion here.